lib/plugins/features/movies/controller.js

```diff
@@ -1,9 +1,56 @@
 'use strict';
 
-const Movie = require('../../../models/movie');
+// Validators
+const Movie     = require('../../../models/movie');
+const MovieList = require('../../../validators/movielist');
+
+// Libraries for SQL requests
+const Boom = require('@hapi/boom');
+const Knex = require('../../../libraries/knex');
 
 exports.create = async (payload) => {
   const movie = await new Movie().save(payload);
 
   return new Movie({ id: movie.id }).fetch();
 };
+
+exports.retrieve = async (query) => {
+  // Perform validation inside method since Hapi doesn't support
+  // validation on GET requests
+  const validation_result = MovieList.validate(query);
```

Comment:
nit: we tend to use camelCase within our node repos, although we do happen to use snake_case when referencing JSON